Mathlib/RingTheory/PrimeSpectrum.lean

```diff
@@ -83,6 +83,13 @@ instance [Subsingleton R] : IsEmpty (PrimeSpectrum R) :=
 
 variable (R S)
 
+/-- The prime spectrum is in bijection with the set of prime ideals. -/
+def equivSetOfIsPrime : PrimeSpectrum R ≃ {I : Ideal R | I.IsPrime} where
```

Comment:
```suggestion
/-- The prime spectrum is in bijection with the set of prime ideals. -/
@[simps]
def equivSetOfIsPrime : PrimeSpectrum R ≃ {I : Ideal R // I.IsPrime} where
```

Reply:
I usually prefer to use coercion from Set; they're defeq but not sure syntactically. Anyway it doesn't matter a lot.

Reply:
Looks like this broke the build, because I'm using the set of maximal ideals is contained in the set of primes; let me change it back.